### ISSUE TITLE
Error starting isochronous mode on powerup

### ISSUE THREAD

**Issue by daliu87** [kemerelab/BehavioralVideo#4]
When connecting to/opening the camera after a power cycle, this error is returned and no image appears:

Source: IidcCameraInternal.cpp(460) Built: Apr  8 2014 21:18:49 - Error starting isochronous stream.
+-> From: Iso.cpp(1821) Built: Apr  8 2014 21:17:39 - Failed isochronous initialization. Error: 0x3.

When trying a second time there's no error and everything loads.

**Comment by ckemere** [comment]
I've noticed this with GigE cameras but not USB. I believe it is also true with the Flycap application.

**Comment by daliu87** [comment]
Yeah just noticed flycap does the same thing, Although don't remember having this issue when testing it way back on 12.04... I'll look into it more.
